Makefile

```diff
@@ -55,6 +59,9 @@ run:
 	  -p EMBEDDINGS_LLM_MODEL_NAME="$(EMBEDDINGS_LLM_MODEL_NAME)" \
 	  -p PROJECT_NAME="$(PROJECT_NAME)" \
 	  -p PROJECT_VERSION="$(PROJECT_VERSION)" \
+	  -p DOWNLOAD_REPO="$(DOWNLOAD_REPO)" \
+	  -p REPO_REMOTE_URL="$(REPO_REMOTE_URL)" \
+	  -p REPO_LOCAL_PATH="$(REPO_LOCAL_PATH)" \
```

Comment:
As a user of this makefile, I will not know the value for this variable. Because it is a file system path that is accessible to the particular Task specific container. Also the name of the project root directory will be hard to guess. for example systemd project, the folder name is systemd-257 

I'd suggest to remove this variable only from this makefile.